executor/aggfuncs/builder.go

```diff
@@ -70,8 +93,8 @@ func buildAvg(aggFuncDesc *aggregation.AggFuncDesc, ordinal int) AggFunc {
 	case aggregation.DedupMode:
 		return nil // not implemented yet.
 
-	// Build avg functions which consume the original data and update their
-	// partial results.
+		// Build avg functions which consume the original data and update their
```

Comment:
remove this tab. so as line 97, 112, 113

Reply:
It's a mistake introduced be Goland, I will fix it.